src/sql-server-util/src/lib.rs

```diff
@@ -62,7 +62,7 @@ impl Client {
     /// IntoFuture and does the default good thing of moving the `Connection`
     /// into a tokio task? And a `.raw()` option that will instead return both
     /// the Client and Connection for manual polling.
```

Comment:
This comment here and above should be updated to reflect new behavior

Reply:
Good catch! Thanks!